meta-balena-common/recipes-containers/resin-supervisor/resin-supervisor/start-resin-supervisor

```diff
@@ -19,19 +19,10 @@ fi
 SUPERVISOR_IMAGE_ID=$(balena inspect --format='{{.Id}}' "$SUPERVISOR_IMAGE:$SUPERVISOR_TAG")
 SUPERVISOR_CONTAINER_IMAGE_ID=$(balena inspect --format='{{.Image}}' resin_supervisor || echo "")
 
-# If self-signed root CA exists, add the CA for the specified registry
+# If self-signed root CA exists, grab it
 if [ -n "$BALENA_ROOT_CA" ]; then
     BALENA_DECODED_ROOT_CA="$(echo "${BALENA_ROOT_CA}" | base64 -d -w0)"
     NODE_EXTRA_CA_CERTS="/etc/ssl/certs/balenaRootCA.pem"
```

Comment:
Is the whole `if [ -n "$BALENA_ROOT_CA" ]; then` needed? The cert should be a part of the system-wide bundle therefore it should not be necessary to use `NODE_EXTRA_CA_CERTS` at all.

Reply:
I think in node's case, you still need to side-load those extra certs even if they are trusted by the system. I can test that, however

Reply:
i tested and it seems once we're trusted by the system, we don't need to explicitly trust in the supervisor.